cogs/tmpc.py

```diff
@@ -434,24 +472,80 @@ async def nomod(self, ctx: Context):
         """
         global moderator
 
-        document = await self.check_tmpc_channel(ctx, is_mod=True)
+        document = await self.check_tmpc_channel(ctx.author, ctx.channel.id, is_mod=True)
         await document.voice.set_permissions(moderator.item, overwrite=None)
         await document.chat.set_permissions(moderator.item, overwrite=None)
         embed: Embed = Embed(title="No Mod",
                              description=f"Mods can't see or join this channel anymore")
         await ctx.reply(embed=embed)
 
-    async def check_tmpc_channel(self, ctx: Context, is_mod: bool = False, everyone: bool = False) \
+    @tmpc.command(pass_context=True,
+                  brief="A user to your temp channel",
+                  help="Default add user to the temp channel, for gaming channel the first argument must be 'False'!\n"
+                       "Member must be invited by mentions.\n"
+                       "Limitations:\n"
+                       "- This command can be used 3/hour/user\n"
+                       "- The invite command is limited to 10 user at once")
+    @cooldown(3, 3600, BucketType.user)
+    async def invite(self, ctx: Context, study_channel: bool):
```

Comment:
Will this really work if no `true` is given?

Reply:
Actually, I expect the command to fail if no truth value was given.

Reply:
If we give study_channel a default value of `True` will it work if we omit the True in the command and just put the mention?

Reply:
No, that will not work.
Let's say this would be your command signature:
```
async def invite(self, ctx: context, study_channel: bool = True, *_):
```
With the command `!tmpc invite (at)user`, the user would be treated as a bool and fail.

Now let's say this would be the signature of your command:
```
async def invite(self, ctx: context, *_, study_channel: bool = True):
```
If you use the command `!tmpc invite user false`
 the `*_` part would take all the arguments, so `study_channel` would still be the default.
 
 The there is now other way expect to separate  methods or drop one type of channel.